src/main/java/org/embulk/input/sftp/SftpFileInputPlugin.java

```diff
@@ -30,11 +30,12 @@ public ConfigDiff resume(TaskSource taskSource,
                              FileInputPlugin.Control control)
     {
         PluginTask task = taskSource.loadTask(PluginTask.class);
+        String lastPath = SftpFileInput.getRelativePath(task, task.getFiles().getLastPath(task.getLastPath()));
         control.run(taskSource, taskCount);
 
         ConfigDiff configDiff = Exec.newConfigDiff();
         if (task.getIncremental()) {
-            configDiff.set("last_path", SftpFileInput.getRelativePath(task.getFiles().getLastPath(task.getLastPath())));
+            configDiff.set("last_path", lastPath);
```

Comment:
When getRelativePath returns null, the plugin should not return {"last_path": null}. It should return empty config_diff. 

```
if (task.getIncremental() && lastPath != null) {
    configDiff.set("last_path", lastPath);
}
```

?